modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
```

Comment:
Forwarding it to 'dom' doesn't work in this case because 'dom' represents the sparse set of indicies, and 'data' is storing the values compactly.